src/lib/components/Timeline.svelte

```diff
@@ -46,6 +60,9 @@
 				{remove}
 				{shift}
 			/>
+			{#if furthest_auto_index === actions.length - i - 1}
+				<hr />
```

Comment:
hr doesn't display anything by default in tailwind, not sure if this is what was intended?

Reply:
Ya, it just adds a gap, which is all that's needed.